userbenchmark/ddp_experiments/__init__.py

```diff
@@ -489,6 +469,222 @@ def get_backend_name(model_args):
     print(job.results())
 
 
+def apply_fsdp_hf_T5_large(model, trainer):
```

Comment:
would it be better to make the majority of this code live in a common 'apply_fsdp' helper, and just make the wrap_policy be determined by the model class or the args string or something?

Reply:
IMO it makes sense to keep this code here - a lot of the configuration here is specific to dynamo, e.g. use_orig_params=True. Does that seem reasonable or do you still think we should move this into some common folder?

Reply:
i was less talking about the folder its in and more the duplication.

I think the only thin you need to customize per model is the wrap policy.  (hopefully).  

use_orig_params=True is going to be a broad requirement for any dynamo FSDP, (and i think @awgu said it will also become the new default at some point)

Reply:
sounds good, I de-duped some of this